firmware/vial/buran/config.h

```diff
@@ -13,13 +13,16 @@
 
 /* --- Matrix definition --- */
 #define MATRIX_ROWS 4 
-#define MATRIX_COLS 11
+#define MATRIX_COLS 12
 
 /* --- Matrix assignment --- */ 
 #define MATRIX_COL_PINS \
-    { GP7, GP8, GP9, GP11, GP10, GP0, GP17, GP18, GP19, GP20, GP21 }
+    { GP7, GP8, GP9, GP11, GP10, GP0, GP17, GP18, GP19, GP20, GP21, GP16}

```

Comment:
oh well I think I've fixed that locally once, yet never pushed it. Sorry and thank you for fixing that!